doc/releases/release-notes-3.4.rst

```diff
@@ -9,6 +9,31 @@ We are pleased to announce the release of Zephyr version 3.4.0.
 
 Major enhancements with this release include:
 
+* Input subsystem: handles input events from various types of input devices and
+  distribute them to other threads in the application.
+* Barrierr API: add architecture agnostic API for data memory barriers.
+* USB Device support:
+
+  * USB device controller API (UDC API)
```

Comment:
nit: dot at the end of the sentences? Not that the rest of the file is coherent about it, but individual sections tend to be coherent within themselves a bit, plus this one stands out

Reply:
yeah, need to sanitize the whole file in one go.